app/rules/page.tsx

```diff
@@ -0,0 +1,116 @@
+                  <li>Don’t be afraid to ask for help from fellow hackers, mentors, or organizers</li>
+                </ul>
+              </li>
+              <li>
+                <span className="font-bold">Build</span>
+                <ul style={{ listStyleType: "square" }} className="ml-8">
+                  <li>Build cool stuff.</li>
+                  <li>If you’re not able to finish your project by the deadline, that’s totally fine.</li>
+                </ul>
+              </li>
+              <li>
+                <span className="font-bold">Share</span>
+                <ul style={{ listStyleType: "square" }} className="ml-8">
+                  <li>Communicate with fellow hackers and teams.</li>
+                  <li>Collaborate with others and help each other out.</li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+          <div className="flex flex-col flex-1 w-full justify-center items-center mt-10">
+            <h3 className="font-TangoSansBold text-2xl pb-5 text-[#EE5000] text-center">Competition Rules</h3>
+            <ul style={{ listStyleType: "circle" }} className="list-spacing">
+              <li>
+                Only current students of Chapman University are eligible to participate in the competition as hackers.
+              </li>
+              <li>Team sizes are limited to a minimum of one person and a maximum of four people.</li>
+              <li>
+                Every team is eligible to submit one project to one of the four development tracks (Artificial
+                Intelligence, Cybersecurity, Healthcare, Sustainability).
+              </li>
+              <li>
+                All team members must be present during event check-in, judging, and the awards ceremony. Leaving the
+                venue during the event to work elsewhere is fine.
+              </li>
+              <li>
+                Teams are encouraged to receive feedback and support from organizers, volunteers, sponsors, and others.
+              </li>
+              <li>
+                All project work must be done within the 48-hour hacking period (
+                {formatHourTo12Hour(hackathonDateInfo.startHour)} on
+                {" " + hackathonDateInfo.month + " " + hackathonDateInfo.startDay + ", " + hackathonDateInfo.year} to
+                {" " + formatHourTo12Hour(hackathonDateInfo.endHour)} on
+                {" " + hackathonDateInfo.month + " " + hackathonDateInfo.endDay + ", " + hackathonDateInfo.year}). Teams
+                can use an idea they had before the event.
+              </li>
+              <li>
+                Teams can work on ideas that have already been done. If somebody wants to work on a common idea they
+                should be allowed to do so and should be judged on the quality of their hack. These days it’s hard to
+                find something that’s fully original and teams might not know an idea has been done before anyway.
```

Comment:
Add needed punctuation and phrase the sentence a little clearer.
```suggestion
                Teams can work on ideas that have already been done. If somebody wants to work on a common idea, they
                should be allowed to do so and should be judged on the quality of their hack. These days, it’s hard to
                find something that’s fully original, and teams might not know that an idea has been done before anyway.
```